testing/sdk_fault_domain.py

```diff
@@ -0,0 +1,52 @@
+'''Utilities related to fault domain information from cloud providers.
+
+************************************************************************
+FOR THE TIME BEING WHATEVER MODIFICATIONS ARE APPLIED TO THIS FILE
+SHOULD ALSO BE APPLIED TO ANY OTHER PARTNER REPOS
+************************************************************************
+'''
+import logging
+
+
+log = logging.getLogger(__name__)
+
+# TODO: use cloud provider library APIs to get list of regions.
+AWS_REGIONS = [
+    'ap-northeast-1',
+    'ap-northeast-2',
+    'ap-south-1',
+    'ap-southeast-1',
+    'ap-southeast-2',
+    'ca-central-1',
+    'eu-central-1',
+    'eu-west-1',
+    'eu-west-2',
+    'sa-east-1',
+    'us-east-1',
+    'us-east-2',
+    'us-west-1',
+    'us-west-2'
+]
+
+# TODO: use cloud provider library APIs to get list of zones.
+AWS_ZONE_SUFFIXES = ['a', 'b', 'c']
```

Comment:
At least a couple regions have `d` and `e`. Can we expand this to a-h for now since it's just a temporary measure?

Reply:
https://github.com/mesosphere/dcos-commons/pull/1855/commits/f074a7ee09e94f2378b69ab958aceb7bf8155e31